packages/@adobe/spectrum-css-temp/components/card/index.css

```diff
@@ -828,7 +828,7 @@ user-provided
     block-size: 100%;
     display: grid;
     grid-template-columns: minmax(0, auto) minmax(30px, 1fr) minmax(0, auto);
-    grid-template-rows: auto var(--spectrum-card-quiet-body-header-margin-top) minmax(var(--spectrum-actionbutton-height), auto) 1fr var(--spectrum-card-body-padding-bottom);
+    grid-template-rows: auto var(--spectrum-card-quiet-body-header-margin-top) minmax(var(--spectrum-actionbutton-height), auto) var(--spectrum-alias-single-line-height) 1fr;
```

Comment:
Made the content section of quiet vertical cards in a CardView have a height equal to a single line height and moved the `1fr` after that so the bottom of the card extends to fill the CardView div (this is specifically for custom layouts where the `itemPadding` provided is much larger than what is needed to render the card's contents).
 
Do we really want to expose `itemPadding` (aka the setting for the vertical height following the Card image) in the GridLayout options? It doesn't feel super useful in the Spectrum CardView case since we limit the content shown in the card to that single line height and if people would use `minSpace` if they wanted to add more gap between the cards vertically. 

I guess it could be useful for people who want to use GridLayout on its own though.

Reply:
I guess quiet vertical wouldn't have a footer, and setting content to a single line height makes sense, do we even need the 1fr when it's in tile? there shouldn't be any space left for it to grow?

Reply:
If the user customizes the `itemPadding` provided to the CardView GridLayout there can be excess space that the card will have to grow into, thus I added the `1fr`